jlm/rvsdg/gamma.hpp

```diff
@@ -23,9 +23,16 @@ class GammaOperation final : public StructuralOperation
 public:
   ~GammaOperation() noexcept override;
 
-  explicit constexpr GammaOperation(size_t nalternatives) noexcept
+  explicit GammaOperation(size_t nalternatives) noexcept
```

Comment:
We generally use `num...` instead of `n...`. Thus, it should be `numAlternatives`

Reply:
done